model.py

```diff
@@ -6194,7 +6195,7 @@ def fetch(cls, _db, lane, type, facets, pagination, annotator,
                 # default page-type feed, which should be cheap to fetch.
                 cls.log.warn(
                     "Could not generate a groups feed for %s, falling back to a page feed.",
-                    lane.identifier
+                    lane.display_name
```

Comment:
I think lane.identifier is still better because display_name can be ambiguous.

Reply:
That's true. What should I use for a WorkList though?

Reply:
Good point. I think using display_name is okay for WorkList, at least for now.